src/components/booklist.css

```diff
@@ -0,0 +1,150 @@
+.Lesson-Panel {
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 4px;
+  width: 80%;
+  background-color: white;
+  padding: 2rem;
+  border: solid 1px whitesmoke;
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.School-of {
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
```

Comment:
- [ ] You might consider group related properties to avoid repetition of the same rule ➿ 
For example, the School-of and Current-Chapter class share three properties. So you can group them as follow 👇🏽 
```
.School-of, .Current-Chapter {
  opacity: 0.5;
  font-size: 0.813rem;
  color: var(--black-two);
}
```